build.sh

```diff
@@ -0,0 +1,149 @@
+#!/bin/bash
+ROOT_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )
+BUILD_DIR="$ROOT_DIR/build"
+RELEASE_DIR="$ROOT_DIR/_RELEASE"
+
+# Removes existing libraries
+rm -f "$RELEASE_DIR/sfml-audio-2.dll"
+rm -f "$RELEASE_DIR/sfml-graphics-2.dll"
+rm -f "$RELEASE_DIR/sfml-network-2.dll"
+rm -f "$RELEASE_DIR/sfml-system-2.dll"
+rm -f "$RELEASE_DIR/sfml-window-2.dll"
+rm -f "$RELEASE_DIR/openal32.dll"
+rm -f "$RELEASE_DIR/libzlib1.dll"
+rm -f "$RELEASE_DIR/OHWorkshopUploader.exe"
+rm -f "$RELEASE_DIR/SSVOpenHexagon.exe"
```

Comment:
Is this needed for Linux builds? Or should we have something similar for Linux that clears up Linux artifacts? I don't think we're going to get `.dll` and `.exe` files on Linux builds.

Reply:
Just had a quick check to be sure and no those artifacts are not made for Linux builds. I will just put those commands for the `"msys"` case.